app/javascript/packs/notificationSubscriptionHandler.js

```diff
@@ -42,23 +42,22 @@ function loadFunctionality() {
   } else {
     updateStatus = target => {
       let payload = '';
-      const shouldUnsubscribeToNotifications = target.classList.contains('selected')
-        || target.classList.contains('selected-emoji');
+      const shouldUnsubscribeToNotifications =
+        target.classList.contains('selected') ||
+        target.classList.contains('selected-emoji');
       const allButtons = document.getElementsByClassName(
         'notification-subscription-label',
       );
       for (let i = 0; i < allButtons.length; i += 1) {
         allButtons[i].classList.remove('selected');
       }
       if (shouldUnsubscribeToNotifications) {
-        const unsubscribeButton = allButtons.namedItem(
-          'unsubscribe',
-        );
+        const unsubscribeButton = allButtons.namedItem('unsubscribe');
         unsubscribeButton.classList.add('selected');
-        payload = unsubscribeButton.dataset.payload;
+        ({ dataset: payload } = unsubscribeButton);
       } else {
         target.classList.add('selected');
-        payload = target.dataset.payload;
+        ({ dataset: payload } = target);
```

Comment:
Similar to above
```suggestion
        ({ payload } = target.dataset)
```

Reply:
Committed the recommended fixes here too.